src/dev/flang/ast/Function.java

```diff
@@ -314,8 +317,18 @@ public AbstractType propagateExpectedType2(Resolution res, AbstractFeature outer
             var f = new Feature(pos(), rt, new List<String>("call"), a, _inherits, _contract, new Impl(_expr.pos(), _expr, im));
             this._feature = f;
 
+            var _inheritsName = Types.FUNCTION_NAME;
```

Comment:
The idea of `_` prefix is that `_` is used only for fields in the surrounding class. This is mainly to avoid confusion in Java constructors that tend to have argument names equal to field names in the class.